src/tabs/tab-header-bar.tsx

```diff
@@ -254,13 +261,20 @@ export function TabHeaderBar({
     function isElementRegistered(element: HTMLElement) {
       return navigationAPI.current?.isRegistered(element) ?? false;
     }
-    function isElementDisabled(element: HTMLElement) {
+    function isElementFocusable(element: HTMLElement) {
```

Comment:
I think the purpose of the code inside the function is not quite apparent and it can be difficult to relate it to the disabled reasons feature. I suggest adding a small code comment to explain the need.